src/solver/ts-generator/thermal.cpp

```diff
@@ -596,11 +597,25 @@ void GeneratorTempData::operator()(Data::Area& area, Data::ThermalCluster& clust
 }
 } // namespace
 
+std::vector<Data::ThermalCluster*> getAllClustersToGen(Data::AreaList& areas,
+                                                       bool globalThermalTSgeneration,
+                                                       bool refreshTSonCurrentYear)
```

Comment:
If `refreshTSonCurrentYear = false` the returned list is empty. Do we really want this ?